apps/dotcom/src/routes.tsx

```diff
@@ -30,28 +29,17 @@ export const router = createRoutesFromElements(
 	>
 		<Route errorElement={<DefaultErrorFallback />}>
 			<Route path="/" lazy={() => import('./pages/root')} />
-			<Route
-				path="/r"
-				loader={() => {
-					const id = 'v2' + nanoid()
-					return redirect(`/r/${id}`)
-				}}
-			/>
-			<Route
-				path="/new"
-				loader={() => {
-					const id = 'v2' + nanoid()
-					return redirect(`/r/${id}`)
-				}}
-			/>
```

Comment:
what's the thinking behind removing these vs keeping them and having them 404 when in an iframe? i think these are super useful for keeping e.g. a bookmark for "new tldraw room"

Reply:
I'd also love to keep this if we can.

It is another vector for how people that embed tldraw could generate new rooms. Spin up a browser, go to `/new` just to generate a new room, then use that new room in an iframe. That said, it's probably just as easy as using the api to get a new slug, so the benefit is quite small.

We did remove the creation of new rooms for slugs that don't exist yet, so we'd have to rework this to go through the `/new-room` api, but it's definitely doable.

Reply:
@steveruizok said in standup that he'll weigh on this to be the decider on this.

I am, personally, a little torn — yes, having a bookmark for this type of thing is definitely cool (we created the domain https://paper.new just to do this ;) 
However, I think the inherent difference with other apps that have this is that we still don't have auth :-/

So, in my mind, this is a temporary removal and once we have an auth story we can reintroduce this.

Reply:
I don't really see how auth comes into this but maybe I'm missing something. Let's see what steve says! Worth calling out that we can keep these but prevent them in an iframe though.